khal/controllers.py

```diff
@@ -27,11 +27,12 @@
 from collections import defaultdict
 from shutil import get_terminal_size
 
-import datetime
+import datetime as dt
```

Comment:
I'd rename variables called `datetime` that shadow this import, rather than `import datetime as`. It will probably make the code more readable to newcomers (or to us in future).